var/spack/repos/builtin/packages/root/package.py

```diff
@@ -164,29 +162,31 @@ class Root(CMakePackage):
             multi=False,
             description='Use the specified C++ standard when building.')
 
-    # ###################### Dependencies ##############################
+    # ###################### Dependencies ######################
 
-    # minimum cmake version required
     depends_on('cmake@3.4.3:', type='build')
     depends_on('pkgconfig', type='build')
 
-    depends_on('lz4',    when='@6.13.02:')
-    depends_on('xxhash', when='@6.13.02:')
-    depends_on('xz')
-    depends_on('pcre')
+    depends_on('blas')
     depends_on('freetype')
+    depends_on('jpeg')
+    depends_on('libice')
     depends_on('libpng')
-    depends_on('ncurses')
+    depends_on('lz4', when='@6.13.02:')  # See cmake_args, below.
+    depends_on('pcre')
+    depends_on('xxhash', when='@6.13.02:')  # See cmake_args, below.
+    depends_on('xz')
     depends_on('zlib')
 
     # X-Graphics
     depends_on('libx11',  when="+x")
     depends_on('libxext', when="+x")
     depends_on('libxft',  when="+x")
     depends_on('libxpm',  when="+x")
+    depends_on('libsm',   when="+x")
 
     # OpenGL
-    depends_on('ftgl',  when="+x+opengl")
+    depends_on('ftgl@2.1.3-rc5',  when="+x+opengl")
```

Comment:
Maybe we should also include this version in the ftgl recipe as the default one.

Reply:
Per #11214, it's the only one mentioned.

Reply:
Nice, did not know about that PR.